src/txkube/test/test_authentication.py

```diff
@@ -68,11 +77,50 @@
 -----END CERTIFICATE-----
 """
 
+# Let hostname u"example.invalid" map to an
+# IPv4 address in the TEST-NET range.
+HOST_MAP = {
+    u"example.invalid.": "192.0.2.2"
+}
+
+def createReactor():
```

Comment:
I guess this should be `create_reactor`.

Reply:
OK